src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -236,15 +236,15 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     {
       if (macroSubstitute(&payload, httpInfo.payload, ce) == false)
       {
-        LM_W(("Bad Input (not sending NotifyContextRequest: payload too large)"));
+        LM_W(("Runtime Error (not sending NotifyContextRequest: payload too large)"));
```

Comment:
99.99%  ... Yeah, can't argue with that.
It's a pity to not give 100% liberty to our users, but in this case you are probably right. Not worth it.